### PR TITLE
Simplify SSH configuration

### PR DESCRIPTION
**Pull request by sestrella**
This PR simplifies SSH configuration by reading SSH authorized keys and user SSH keys from Ansible variables instead of files.

### CODE REVIEW

**Comment by goetzc** [comment]
We use already [this tag](https://github.com/stackbuilders/sb-debian-base/blob/master/tasks/main.yml#L33) :)

**Comment by lpaulmp** [comment]
@sestrella Looks good

**Comment by sestrella** [comment]
@goetzc, @lpaulmp could you guys take a look, please.

File: tasks/bootstrap-deploy-user.yml [hunk 1]
```diff
@@ -25,20 +25,15 @@
...
 
-- name: Add SSH keys for GitHub's "Deploy keys"
-  template:
-    src:  "{{ item.src }}"
-    dest: "{{ item.dest }}"
-    mode: "{{ item.mode }}"
+- name: Add deploy user private key
+  copy:
+    dest:  "{{ sb_debian_base_deploy_user_dir }}/.ssh/id_rsa"
+    content: "{{ sb_debian_base_deploy_user_private_key }}"
```
Comment (lpaulmp) [inline tasks/bootstrap-deploy-user.yml]: :+1:

Comment (sestrella) [inline tasks/bootstrap-deploy-user.yml]: @lpaulmp  I'm not sure about defining default values for `sb_debian_base_admin_user_authorized_keys` and `sb_debian_base_deploy_user_authorized_keys` that may lead to potential SSH lockout problems, if you disable SSH password login without setting up some authorized keys first. I would rather raise and error if `sb_debian_base_admin_user_authorized_keys` or `sb_debian_base_deploy_user_authorized_keys` are not defined, I think is a little bit more safe. What do you think? \cc @goetzc

Comment (goetzc) [inline tasks/bootstrap-deploy-user.yml]: I agree, we shouldn't define any SSH keys as default. When those variables are not defined, Ansible will interrupt and exit with an error, we don't need to do any further checks here.

**Pull request merged**